python/ray/workflow/tests/test_basic_workflows_2.py

```diff
@@ -166,15 +166,6 @@ def wait(obj_ref):
     # Once job finished, we'll be able to get the result.
     lock.release()
     assert 4 == ray.get(output)
-
-    # Here sometimes inner will not be generated when we call
-    # run_async. So there is a race condition here.
-    try:
-        v = ray.get(inner)
-    except Exception:
-        v = None
-    if v is not None:
-        assert 2 == 20
```

Comment:
Should be `2 == v`? But doesn't seem needed to check in any case.

Reply:
We need this one. v == 2 should be the right onw.